src/ucs/sys/topo/base/topo.c

```diff
@@ -334,6 +362,27 @@ static void ucs_topo_pci_root_distance(const char *path1, const char *path2,
                                   (19200.0 * UCS_MBYTE) / path_distance);
 }
 
+static void ucs_topo_common_numa_node_distance(ucs_sys_dev_distance_t *distance)
+{
+    distance->latency   = 300e-9;
```

Comment:
does it depend on the platform?

Reply:
I think yes, I does. But during the investigation I measured only BW. I can try to measure it too.